### ISSUE TITLE
VOLK_IMPLEMENTATION is also defined after including volk.h

### ISSUE THREAD

**Issue by ohmaya** [PacktPublishing/The-Modern-Vulkan-Cookbook#5]
https://github.com/zeux/volk
"In exactly one source file, define VOLK_IMPLEMENTATION before including volk.h."

Well, VOLK_IMPLEMENTATION is defined twice in this book's code base:

- Context.cpp
  before including volk.h
- Common.cpp
  **after including volk.h, actually no effect, I guess**

Thanks.

**Comment by preetishkakkar** [comment]
Yes, that seems like unnecessary, but I don't think it has any side effects. We shall remove it.

**Comment by ohmaya** [comment]
The only side effect is confusion, I believe. : )

**Comment by preetishkakkar** [comment]
Agreed. I will remove it
